eZ/Publish/Core/MVC/Symfony/Routing/UrlAliasRouter.php

```diff
@@ -296,6 +296,14 @@ public function getRouteCollection()
      */
     public function generate($name, $parameters = [], $referenceType = UrlGeneratorInterface::ABSOLUTE_PATH)
```

Comment:
AFAICS you need to align the prototype with Symfony5:
```php
public function generate(string $name, array $parameters = [], int $referenceType = self::ABSOLUTE_PATH)
```

Reply:
Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/3000/commits/86089127dac95c5144cb191da4a8ba35280d451e